tock/projects/models.py

```diff
@@ -23,6 +23,8 @@ class Project(models.Model):
     name = models.CharField(max_length=200)
     iaa = models.CharField(max_length=200, blank=True)
     agency = models.ForeignKey(Agency)
+    description = models.TextField(blank=True, null=True)
+    billable = models.BooleanField(default=False)
```

Comment:
Shouldn't we be able to determine the billable status of a project via the existence (or non-existence) of `iaa`? Then we could eliminate the `billable` field.